csunplugged/static/scss/programming-editor.scss

```diff
@@ -20,51 +105,250 @@ body,
 }
 
 .programming__info-area {
-  overflow-y: scroll;
+  width: 100%;
   height: 100%;
-  padding: 1.25rem !important;
+  padding: 20px 10px;
+  padding-left: 20px;
+  
+  .programming__info-area-content {
+    background-color: white;
+    overflow-y: auto;
+    padding: 1.25rem !important;
+    height: 100%;
+    width: 100%;
+    border: 1px solid grey;
+    border-radius: 15px;
+
+    h1 {
+      font-size: 2.0rem;
+    }
+  }
+
+  // Removing the testing examples section from content (temporary).
+  #testing-examples, #testing-examples + p, #testing-examples + p + table, #requirement {
+    display: none;
+  }
+
 }
 
 .programming__results {
-  padding: 0px !important;
+  padding: 20px 10px !important;
+  padding-right: 20px !important;
+
+  .results-table {
+    border: 1px solid grey;
+    border-radius: 15px;
+    background-color: white;
+
+    .table {
+      margin-bottom: 0px;
+    }
+  }
+}
+
+.pii-block-heading {
+  color: #292b2c;
+  font-size: 1.1em;
+  font-weight: bolder;
+  font-family: "Noto Sans", sans-serif;
+  margin-bottom: 10px;
 }
 
 .programming__editor {
   height: 100%;
-  padding: 0px;
-
+  padding: 20px 10px;
+  display: flex;
+  flex-direction: column;
+  
   .CodeMirror {
+    flex-grow: 1;
     margin: 0px;
-    height: 95%;
+    border: 1px solid grey;
+    border-radius: 15px;
   }
 
   .programming__submit {
-    height: 5%;
-    background-color: #d6c1e1ff;
+    height: 55px;
 
     .code_running_spinner {
       display: none;
-      color: white;
+      color: #5a9fffff;
       margin-right: 0.625rem;
     }
 
-    #editor_run_button {
-      border: none;
-      background-color: #06a284ff;
-      color: white;
+    .programming__submit_options {
       height: 100%;
-      width: 20%;
-      transition: ease-in;
+      align-items: flex-end;
+    
+      .editor_button {
+        border: none;
+        background-color: #5a9fffff;
+        border: 2px solid #5a9fffff;
+        color: white;
+        height: 80%;
+        padding: 0px 20px;
+        margin: 0px 10px;
+        border-radius: 20px;
+        transition: ease-in;
+        min-width: 100px;
+      }
+
+      .editor_button:hover, #download_button:hover {
+        cursor: pointer;
+        background-color: #1b7bffff;
+        border: 2px solid #1b7bffff;
+        color: white;
+      }
+
+      #download_button {
+        border: 2px solid #5a9fffff;
+        color: #5a9fffff; 
+        background-color: transparent;
+      }
+
+      #editor_run_button:disabled {
+        background-color: #a4caffff;
+        border-color: #a4caffff;
+        cursor: normal;
+      }
+    }
+  }
+}
+
+.programming__results {
+  display: flex;
+  flex-direction: column;
+  max-height: 100%;
+
+  .results-table {
+    flex-shrink: 1;
+    overflow-y: auto;
+  }
+}
+
+// For small devices
+@media (max-width: 768px) { 
```

Comment:
Just a note that might be handy for the future.. we use [bootstraps sass mixins](https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints) for things like this :)

Reply:
Assuming you are meaning something like `@include media-breakpoint-down(sm)`, that does look a lot nicer!

However, I just tried to add it in and it errors saying `Error: no mixin named media-breakpoint-down`. I've looked this up and it appears that the following imports need to occur

```
@import "~bootstrap-4.0.0/scss/_functions.scss";
@import "~bootstrap-4.0.0/scss/_variables.scss";
@import "~bootstrap-4.0.0/scss/mixins/_breakpoints.scss";
```

But I can't seem to find these imports anywhere in the project on a global search. Any ideas?

Reply:
Does putting this at the top of your file work? If not, don't worry about it :)

```
@import "node_modules/bootstrap/scss/functions";
@import "node_modules/bootstrap/scss/variables";
@import "node_modules/bootstrap/scss/mixins";
```

Reply:
Yep, that worked, thanks!

Reply:
Sweet! Now you can replace line 233 with this: `@include media-breakpoint-down(sm) { ... }`